6MI0800226/week1.cpp

```diff
@@ -0,0 +1,114 @@
+#include <iostream>
+#pragma warning(disable: 4996)
+struct transition
+{};
+
+
+//zad 2
+struct pair
+{
+    int firstInt;
+    int secondInt;  
+};
+void inputIntsOfPair(pair& pair1)
+{
+    std::cout << "First integer: ";
+    std::cin >> pair1.firstInt;
+    std::cout << "Second integer: ";
+    std::cin >> pair1.secondInt;
+}
+bool isBigger(const pair& firstPair, const pair& secondPair)
+{
+    return firstPair.firstInt > secondPair.firstInt ||
+        (firstPair.firstInt == secondPair.firstInt && firstPair.secondInt > secondPair.secondInt);
+}
+pair biggestPair(const pair* pairs, size_t size)
+{
+    size_t indexOfBiggestPair = 0;
+    for (size_t i = 1; i < size; i++)
+    {
+        if (isBigger(pairs[i], pairs[indexOfBiggestPair]))
+            indexOfBiggestPair = i;
+    }
+    return pairs[indexOfBiggestPair];
+}
+void swap(pair& pair1, pair& pair2)
+{
+    pair temp = pair2;
+    pair2 = pair1;
+    pair1 = temp;
+}
+void sortPairs(pair* pairs, size_t size)
+{
+    for (size_t i = 1; i < size; i++)
+    {
+        pair biggestPair1 = biggestPair(pairs + i, size - i);
+        if (isBigger(biggestPair1, pairs[i - 1]))
+            swap(biggestPair1, pairs[i - 1]);
+        //std::cout << (pairs + i)->secondInt << " " << size - 1;
+    }
+}
+
+
+
+//zad 3
+enum class classes
+{
+    first,
+    second,
+    third,
+    fourth
+};
+struct Student
+{
+    char* name;
+    size_t facultyNumber;
+    classes studentClasses;
+    float grade;
+};
+void classesToChar(char* classesChar, const classes& cl)
+{
+    if (cl == classes::first)
```

Comment:
You could've used a `swtich/case` statement too (see: [example](http://www.java2s.com/ref/cpp/cpp-enum-value-on-switch-statement.html))